x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_overview/overview_card.tsx

```diff
@@ -42,8 +49,9 @@ export const OverviewCard: FunctionComponent<Props> = ({
             justifyContent="spaceBetween"
             wrap={true}
             alignItems="center"
+            // euiButtonHeightSmall
             css={css`
-              min-height: ${euiThemeVars.euiButtonHeightSmall};
+              min-height: ${euiTheme.size.s};
```

Comment:
todo verify

Reply:
@mattkime `euiButtonHeightSmall` maps to 32px, so the equivalent is `size.xl`.
`size.s` is 8px

```suggestion
              min-height: ${euiTheme.size.xl};
```

Reply:
thank you!